### DISCUSSION TITLE
Permission error on bootstraping board remotly

### DISCUSSION THREAD

**Discussion by Mdfayaz003** [labgrid-project/labgrid#1107] [Q&A] [answered]
As I discussed on, I was trying to bootstrap a remotely connected NXP board where the coordinator and exporter running on one PC and client on different PC. 

I got a `Permission error` since it was trying to create files on `/var/cache` location. I tried changing mode and ownership but it resulted on same error.
```
sudo chmod a+x /var/cache
sudo chown -R $USER:$USER /var/cache
```

Can anyone help me out with this problem?

**Comment by jluebbe** [comment, marked as answer]
Please take a look at https://labgrid.readthedocs.io/en/stable/getting_started.html#systemd-files. The `systemd-tmpfiles` should setup the correct directories.

**Reply by Mdfayaz003** [reply]
Thanks @jluebbe. Now the error is solved and achieved flashing images on board connected remotely
